config/broker/broker-admin/role.yaml

```diff
@@ -52,9 +52,6 @@ rules:
       - delete
   - apiGroups:
       - discovery.k8s.io
-    resources:
-      - endpointslices
-      - endpointslices/restricted
```

Comment:
Why was this change made?

Reply:
seems like side-effect of a rebase....Will check